applications/nrf_desktop/src/modules/smp.c

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <zephyr/types.h>
+
+#include <mgmt/smp_bt.h>
+#include "img_mgmt/img_mgmt.h"
```

Comment:
why mixing header inclusion styles? are this taken from different scopes?

Reply:
Changed, it was leftover from previous implementation